net/magic-wormhole/Portfile

```diff
@@ -31,7 +31,7 @@ checksums           rmd160  bbb9d49f4520e5168f95b35264a4fce7ce4732a7 \
                     sha256  1b0fd8a334da978f3dd96b620fa9b9348cabedf26a87f74baac7a37052928160 \
                     size    262269
 
-python.default_version  38
+python.versions     38 39 310
```

Comment:
this will not work; since the port doesn't start with `py-` it will not generate subports for the different Python versions you supply here. In such a case the only way is to use `variants` if you really want to have it to work with different Python versions; however, in most case (as here as well) I don't see a reason why you would want do to that anyway. We typically stick to the "MacPorts default Python version" as defined in the `python` PG which is currently 3.9

Reply:
Good point here @reneeotten, I missed this.